### DISCUSSION TITLE
Really liking Friesach

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#287] [General]
Originally posted 2019-10-20 00:36:12.189000 by mashaffer in GrandOrgue discussion forum on SourceForge<br><br>I downloaded Friesach (https://www.piotrgrabowski.pl/friesach.html) and I think it may be my favorite GrandOrgue disposition. Even the tremulants are quite usable just as they are. The voices speak with good clarity without being too &quot;in your face&quot; and with none of the &quot;honkiness&quot; one sometimes gets (especially in the prinipals). The 32&quot; pedal stop is not as clear (distinct) as those in some jOrgan dispositions I have used but it is quite useful. I imagine that the 32s are the most difficult to record and process and also difficult for the pipe organ builders to make clean in the first place.

In short I highly recommend downloading it and taking it for a spin if you haven&#039;t already..

**Comment by larspalo** [comment]
Originally posted 2019-10-21 02:39:00.053000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>It&#039;s my number one sampleset - i don&#039;t even bother with any others, none come up to at it all that i&#039;ve come across.  Thats not to say there could be improvements but for the changes available and ammendments on offer within GrandOrgue its fine!

I wish Piotr had time to go over some older ones he&#039;s done and redo them cleanly... with all the techniques he&#039;s acquired.  Also some closer mic with some of the samplesets including Friesach would be awesome but he&#039;s done a tremendous job and service to the hobby.

**Comment by larspalo** [comment]
Originally posted 2019-10-21 18:18:53.030000 by leobsb in GrandOrgue discussion forum on SourceForge<br><br>I like Friesach a lot too. However, the sound I hear is not very good. In. Some records hear a sharp noise in the middle notes. When I use TUTTI, the sound sounds distorted ...

I use GO on a lenovo i7 and 16g ram notebook and a Presonus Audiobox 22vsl interface. The sample is loaded at 16 bits and 1024.

What could I do to fix these issues?

**Comment by larspalo** [comment]
Originally posted 2019-10-21 23:08:56.099000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>1, 24bit
2, load organ and delete turn off all the stops that you dont need to fit into 16gb ram ie 16&#039; pipes in manuals and 32&#039; pedals also turn off all stoos in pedals 8&#039; 4&#039; etc  just leave the 2 flute16&#039; if there are other stops that you dont want turn them iff instead just to get the sample fully loaded in &quot;organ setup&quot;

3, run in polyphase not linear in audio settings

4, alternatively run 1 release sample instead of all to get sample size down if you eant all the stoos

5, set output volume to -5 to 8db at least you dont ever want to see red in thr output meters you are most likely distorting the outputs if all the stops are running out of 1 x stereo channel only

6,  save and reload sampleset and update chache if you use it, that should drasticly inprove the sound for you let us know what you hear after all that....


i run all balanced xlr out of 8 channels with sll the stops spread  across them at 24bit and all as i have 32gb to 96gb of ram depending on machine however my macbook 13&quot; only has 16gb for a quick portable setup.

**Comment by larspalo** [comment]
Originally posted 2019-10-21 23:21:06.809000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>also try dropping buffer to 512 that will halve your latency make sure you set latency requested in audio settings sound to 1 and check sound output thrn you can adjust up to to 10ms or 20ms

also make sure to turn off release sample  scaling and you can try turning off the detune on release one as well if its detuning to much on each note....

after all that if it still doesnt sound awesome you maybe hearing the actual recordings as thry sre in the irgan... not much you can do about that but definately running at the recorded but and resolution is important ie 24bit and 48khz   sounds lije your on windows make sure you set sound card to 24bit and 48khz in the separate control panel tab to match

**Comment by larspalo** [comment]
Originally posted 2019-10-22 01:03:59.415000 by leobsb in GrandOrgue discussion forum on SourceForge<br><br>Thank you very much for the tips, Matt!

If I understand correctly, my main problem is low memory... Sorry for my ignorance, but how do I turn off a stop? Deleting directly in the organ file before loading?

**Comment by larspalo** [comment]
Originally posted 2019-10-22 03:21:11.831000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>your main problem is that you are loading the sampleset at 16bit to get it to fit into your 16gb of ram - you need to run the sampleset at its recorded resolution 24bit 48khz so the only way for you to do that is by reducing the amount of stops loaded.  You do that in Organ Settings from one of the main GO menus go into there and click on organise or something like that and turn off ie &quot;don&#039;t load&quot; all the stops/sounds that you don&#039;t need 32&#039; some 16&#039;  you will need to turn off all the ones i mention above... for quick and easy just turn off a whole division and add back ones that you want.  You should then be able to load it into memory have no other programs running and turn off any features background apps running to get as much ram as necessary freed up.  You need to limit the size of the sampleset to 14.5-15gb depending on your OS system and what it needs to operate.

**Comment by larspalo** [comment]
Originally posted 2019-10-22 12:37:44.341000 by tbpr in GrandOrgue discussion forum on SourceForge<br><br>You can also choose to compress the samples. This saves a lot of memory. That will enable you to save more stops. Also you should not load any of the noises. They don&#039;t consume a lot of memory, but every little bit helps. (I don&#039;t like the noises anyway.)

Edited to add: Also, you might find loading the sample sets at 20 bits to be perfectly acceptable.

**Comment by larspalo** [comment]
Originally posted 2019-10-25 02:50:05.253000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>Using compression however will put you back into using Lenear which doesn&#039;t sound very good.  Better if audio is paramount to use polyphase and disable not needed stops and background noises...!

**Reply by larspalo** [reply]
Originally posted 2019-10-25 13:17:24.657000 by tbpr in GrandOrgue discussion forum on SourceForge<br><br>Really? Maybe my hearing isn&#039;t as good as yours. I don&#039;t think it sounds bad.

**Comment by larspalo** [comment]
Originally posted 2019-10-25 22:16:53.657000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>ok out another way:  neither bad but they have different jobs and ways  going about similiar things!

However, linear is cleaner and better in bass and polyohase is better and less distortion for treble.  The OP was hearing artifacts and &quot;noise&quot; so that instantly told me to use polyphase for better treble and cleaner upper sound.

i have very sensitive ears its a joy but a curse sometimes.... for eg i can spot a mp3 a mile away! even a &quot;good&quot; one...

**Reply by larspalo** [reply]
Originally posted 2019-10-25 22:57:59.066000 by [Nick Appleton](https://github.com/nickappleton) in GrandOrgue discussion forum on SourceForge<br><br>Linear interpolation introduces audible artefacts into the audio when resampling is occurring. The polyphase interpolator improves the suppression of these artefacts substantially. Resampling occurs whenever GrandOrgue does any sort of pitch shifting of a sample - if the sample is not being pitch-shifted dramatically, the impact is reduced but still present. It’s the reason that I added the polyphase filter to GrandOrgue (and it is still not a particularly good one).

Linear interpolation artefacts are easier to hear when playing high-pitched samples. The resampling will introduce new frequencies above and below the fundamental of the note being played that are not harmonically related.

This is a technical blog I wrote a while ago which outlines the signal processing behind linear interpolation: https://www.appletonaudio.com/blog/2015/real-time-re-sampling-and-linear-interpolation/ - it’s probably too technical in this forum, but this picture from it is relevant: http://www.appletonaudio.com/wp-content/uploads/2015/11/toneinterp.png Input signal is a bunch of tones separated logarithmically in frequency. It gets resampled using both linear and a polyphase interpolator. The blue trace shows the linear interpolated result, the green one is the polyphase interpolated result. The blue one has lots of “extra” stuff generated in it (which is absolutely audible) and the green one does not.

Nick

&gt; On 26 Oct 2019, at 9:16 am, Matt &lt;mattrixed@users.sourceforge.net&gt; wrote:
&gt; 
&gt; ok out another way:  neither bad but they have different jobs and ways  going about similiar things!
&gt; 
&gt; However, linear is cleaner and better in bass and polyohase is better and less distortion for treble.  The OP was hearing artifacts and &quot;noise&quot; so that instantly told me to use polyphase for better treble and cleaner upper sound.
&gt; 
&gt; i have very sensitive ears its a joy but a curse sometimes.... for eg i can spot a mp3 a mile away! even a &quot;good&quot; one...
&gt; 
&gt; 
&gt; ---
&gt; 
&gt; [Really liking Friesach](https://sourceforge.net/p/ourorgan/discussion/962124/thread/97bfc2a89c/?limit=25#069a)
&gt; 
&gt; 
&gt; ---
&gt; 
&gt; Sent from sourceforge.net because you indicated interest in &lt;https://sourceforge.net/p/ourorgan/discussion/962124/&gt;
&gt; 
&gt; 
&gt; 
&gt; To unsubscribe from further messages, please visit &lt;https://sourceforge.net/auth/subscriptions/&gt;

**Comment by larspalo** [comment]
Originally posted 2019-10-26 00:30:49.868000 by mattrixed in GrandOrgue discussion forum on SourceForge<br><br>Thanks Nick, i sorta knew some of it and didnt know how to explain it.

Every new installation i auto click on polyphase.

i suppose if you could you would have further improved the filter?
